.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,36 +6,34 @@ labels: bug, p1
 assignees: ''
 
 ---
+<!--Fields in **bold** are REQUIRED, fields in *italics* are OPTIONAL -->
 
-**Describe the bug (REQUIRED)**
-A clear and concise description of what the bug is.
+**Describe the bug**
+<!-- A clear and concise description of what the bug is. -->
 
+**Reproduction Steps**
+<!-- Steps to reproduce the behavior -->
 
-**Reproduction Steps (REQUIRED)**
-Steps to reproduce the behavior:
-1. Typed the following: '...'
+**Outputted Messages**
+<!-- Provide the full message log if possible or a subset that includes a few lines before the failure -->
 
-
-**Outputted Messages (REQUIRED)**
-Provide the full message log if possible or a subset that includes a few lines before the failure:
-
-
-**Version information (please complete the following, default in brackets): (REQUIRED)**
- - LD_Cool-P version: [e.g. `0.13.0`] 
- - ReQUIAM branch info: [e.g., `release/v0.13.0`]
+**Version information (please complete the following, default in brackets)**
+ - LD_Cool-P version: [e.g. `0.17.0`] 
  - `pandas` version: [e.g., 1.0.2]
  - `requests` version: [e.g., 2.22.0]
  - `numpy` version: [e.g., 1.17.4]
+ - `tabulate` version: [e.g., 0.8.3]
  - `jinja2` version: [e.g., 2.11.2]
+ - `html2text` version: [e.g, 2020.1.16]
```

Comment:
After a bit of thought, I'm finding this to be a bit cumbersome to maintain. It's a good idea, but I don't want it to be burdensome. So where's what I think, we should consider just linking the requirements.txt for either the tagged version or the branch where the problem occurred. That way we can still see it. Here's the suggested revision
```suggestion
**Version information**
<!-- Update with tagged version or branch full name-->
[requirements](https://github.com/ualibraries/LD_Cool_P/blob/v<version>/requirements.txt)
[requirements](https://github.com/ualibraries/LD_Cool_P/blob/<branch>/requirements.txt)
```